tests/ert_tests/analysis/test_es_update.py

```diff
@@ -173,48 +167,6 @@ def test_update_snapshot(setup_case, module, expected_gen_kw):
     assert target_gen_kw == pytest.approx(expected_gen_kw)
 
 
-@pytest.mark.integration_test
-def test_that_posterior_has_lower_variance_than_prior_with_init_files(copy_case):
```

Comment:
Could we convert this test to just sample the parameters normally? I think this is the only integration test we have that actually checks that the update is correct. Probably enough to replace: `copy_case("local/externally_sampled_params")` with: `copy_case("poly_example")`, but think @dafeda can confirm?

Reply:
Good idea, I agree that we should keep the tests but use poly_example instead.